plugin-management-library/src/test/java/io/jenkins/tools/pluginmanager/impl/PluginManagerTest.java

```diff
@@ -76,26 +73,7 @@ public void startTest() {
 
         PluginManager pluginManager = new PluginManager(config);
 
-        PluginManager pluginManagerSpy = spy(pluginManager);
-
-        doNothing().when(pluginManagerSpy).createRefDir();
-        VersionNumber versionNumber = new VersionNumber("2.182");
-        doReturn(versionNumber).when(pluginManagerSpy).getJenkinsVersionFromWar();
-        doNothing().when(pluginManagerSpy).getUCJson(versionNumber);
-        doReturn(new HashMap<>()).when(pluginManagerSpy).getSecurityWarnings();
-        doNothing().when(pluginManagerSpy).showAllSecurityWarnings();
-
-        doReturn(new HashMap<>()).when(pluginManagerSpy).bundledPlugins();
-        doReturn(new HashMap<>()).when(pluginManagerSpy).installedPlugins();
-        doReturn(new HashMap<>()).when(pluginManagerSpy).findPluginsAndDependencies(anyList());
-        doReturn(new ArrayList<>()).when(pluginManagerSpy).findPluginsToDownload(anyMap());
-        doReturn(new HashMap<>()).when(pluginManagerSpy).findEffectivePlugins(anyList());
-        doNothing().when(pluginManagerSpy).listPlugins();
-        doNothing().when(pluginManagerSpy).showSpecificSecurityWarnings(anyList());
-        doNothing().when(pluginManagerSpy).checkVersionCompatibility(any(), anyList());
-        doNothing().when(pluginManagerSpy).downloadPlugins(anyList());
-
-        pluginManagerSpy.start();
+        pluginManager.start();
```

Comment:
will this cause the test to be reliant on the network and the update center being available?

Reply:
You're right. I try to improve the test.

Reply:
@stefanbirkner Hi, are you still interested in finishing this pull request?